src/wfdir.c

```diff
@@ -1318,6 +1318,8 @@ ChangeDisplay(
 
    case WM_CREATE:
 
+      lpStart = NULL;

```

Comment:
Why does this satisfy the SdlCheck?
lpStart is used above many times

Reply:
The compiler understands codeflow, it just doesn't understand the relationship between multiple variables.  Here, the outer switch statement has three case branches.  The first two initialize `lpStart` on line 1064.  These also have several goto statements that can jump into the `WM_CREATE` case, but only after `lpStart` is initialized.  In the `WM_CREATE` case (without any goto) it's initialized because `bCreateDTABlock` must be `TRUE` - but that exceeds the compiler's reasoning capabilities.  Without doing something to cover `WM_CREATE`, the compiler complains at line 1478 because it doesn't know that it must have taken the `bCreateDTABlock` branch above.

Reply:
I see. Thx. 
To make the intention of this initialization more obvious to the human reviewer, please initialize it in line 1000 with the declaration, which then has the same effect

Reply:
The earlier these initializations occur, the less effective the warning is.  The warning can catch when values are used that are not initialized; it cannot catch values that are initialized with an invalid value such as NULL.  Keeping it here means that if any code above it tries to use the variable without initializing it, it can still be caught.  The frustrating part is most of the time this isn't possible, which limits the value of the warning.